spec/lib/submission_builder/ty2024/states/nj/nj_return_xml_spec.rb

```diff
@@ -26,10 +26,12 @@
       end
 
       context "with one dep" do
-        let(:intake) { create(:state_file_nj_intake, :df_data_one_dep, municipality_code: "0101") }
+        let(:intake) { create(:state_file_nj_intake, :df_data_one_dep) }
         it "does not error" do
           builder_response = described_class.build(submission)
           expect(builder_response.errors).not_to be_present
+          expect(builder_response.document.at("WagesSalariesTips").text).not_to eq(nil)
+          expect(builder_response.document.at("NewJerseyTaxableIncome").text).not_to eq(nil)
```

Comment:
question: the description of this says "does not error" but i can't tell why it cares about these two fields specifically (and just that they are not nil?)

Reply:
Ah, good catch. We can probably delete this. We wanted some way to validate that it was building "as expected" - ie, that if there were W2 wages in the original DF XML, there should be output elements in the state return that reflect those W2 wages. Another test to ensure that they were actually getting synced and used.

I think we're now testing this sufficiently in other places though